kubeadm/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.fedoraproject.org/fedora:rawhide
+MAINTAINER "Jason Brooks" <jbrooks@redhat.com>
+
+ENV container=docker
```

Comment:
I don't see a reference to this used in the system container itself. It may be worth putting a comment above it explaining what it's used for.